bool has_trash
float32 dist_to_trash
float32 err_to_center
